sdk/nodejs/src/client.ts

```diff
@@ -296,6 +303,27 @@ export class DevCycleClient {
         this.eventQueue.queueEvent(populatedUser, event)
     }
 
+    private trackSDKConfigEvent(
+        url: string,
+        responseTimeMS: number,
+        res: Response,
+    ): void {
+        this.eventQueue.queueEvent(
+            DVCPopulatedUserFromDevCycleUser({ user_id: this.clientUUID }),
+            {
+                type: 'sdkConfig',
+                target: url,
+                value: responseTimeMS,
+                metaData: {
```

Comment:
Do we want to include request header data too?

Reply:
Specifically etag/LM

Reply:
that's in the line below...

Reply:
Those are the response headers I thought, I'm saying send them both

Reply:
ahh good idea